addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -279,6 +271,20 @@ function factory(dependencies) {
             this._toggleVideoBroadcast({ type: 'user-video' });
         }
 
+        async unDeafen() {
+            await this._setDeafState(false);
+        }
+
+        async unMute() {
+            if (this.audioTrack) {
+                this.currentRtcSession.updateAndBroadcast({ isMuted: false });
+                await this._updateLocalAudioTrackEnabledState();
+            } else {
+                // if we don't have an audioTrack, we try to request it again
+                await this.updateLocalAudioTrack(true);
+            }
+        }
```

Comment:
At a glance of the algorithms between `(un)deafen` and `(un)mute`, they have similar shape.
However, the split in function and how they are called give the wrong impression they are more different than they really are.

It's worth cleaning so they look more alike as they should.